client/src/components/categorical/category/index.js

```diff
@@ -198,6 +202,15 @@ class Category extends React.Component {
       globals.categoryDisplayStringMaxLength
     );
 
+    const handleCategoryClick = () => {
```

Comment:
why take the performance hit of doing this on each render?  You could make it an instance value and have same effect, with less overhead.

Reply:
The function was originally an arrow function in render/props and I refactored without thinking of performance enhancements.  I'll add it as a class property 👍

Reply:
cool - it will need to be an assigned instance property, so that it gets the bind for free.

Reply:
My terminology knowledge is failing me. Could you elaborate on "assigned instance property"? I typically get my bind by defining the prop as `prop={()=>functionName()}`

Reply:
it should be unique to each instance, rather than being in the prototype.

In other words, create it thusly:
```
class Foo {
   // ...
   handleCategoryClick = (e) => { /* ... */ };
}
```

Rather than the more typical:
```
class Foo {
  // ...
  handleCategoryClick(e) { /* ... */ }
}
``

The former will have `this` bound to the instance, the latter will not.